ticket_booking_app/src/main/java/com/mordor/model/dto/ReservationConfirmationDTO.java

```diff
@@ -0,0 +1,54 @@
+package com.mordor.model.dto;
+
+import java.net.URL;
+import java.time.Instant;
+
+public class ReservationConfirmationDTO {
+	
+	private double totalAmount;
+	
+	private Instant expirationTime;
+	
+	private URL confirmationLink;
+
+	public double getTotalAmount() {
+		return totalAmount;
+	}
+
+	public void setTotalAmount(double totalAmount) {
+		this.totalAmount = totalAmount;
+	}
+
+	public Instant getExpirationTime() {
+		return expirationTime;
+	}
+
+	public void setExpirationTime(Instant expirationTime) {
+		this.expirationTime = expirationTime;
+	}
+
+	public URL getConfirmationLink() {
+		return confirmationLink;
+	}
+
+	public void setConfirmationLink(URL confirmationLink) {
+		this.confirmationLink = confirmationLink;
+	}
+
+    @Override
+    public boolean equals(Object o) {

```

Comment:
Ładnie zaimplementowany equals! Na przyszłość polecam Lombok'a, który sam Ci to wygeneruje :)